### DISCUSSION TITLE
Running multiple instances of code-server on one vm

### DISCUSSION THREAD

**Discussion by dvyas1** [coder/code-server#2097] [Q&A] [answered]
Is it possible to run multiple instances of code-server on same virtual machine? Other than port conflict that can be mitigated by using different port for each instance, do you see any other issues?

**Comment by bpmct** [comment, marked as answer]
Yes - totally possible! Like you said, you can just run a different port for each instance. You can also specify different config files for each instance as well.

If you are looking to do something more robust like running code-server instances on K8s, docs for that are also being worked on (#2048, #2003).
